smart-contracts/test/Unlock/upgrades/v0ToLatest.js

```diff
@@ -8,7 +8,7 @@ ZWeb3.initialize(web3.currentProvider)
 const UnlockV0 = Contracts.getFromNodeModules('unlock-abi-0', '../../Unlock')
 const PublicLockV0 = require('public-lock-abi-0/abi_V0')
 
-const UnlockV1 = Contracts.getFromLocal('Unlock')
+const UnlockLatest = Contracts.getFromLocal('Unlock')
```

Comment:
`getFromLocal` should always be the latest version of Unlock, correct?

Reply:
I should probably modify the naming for publicLock as well to reflect the fact that we're using the latest, not necessarily v1.